### ISSUE TITLE
Should boolean Chapel environment variables all use the same values?

### ISSUE THREAD

**Issue by ben-albrecht** [chapel-lang/chapel#16382]
To my knowledge, the boolean Chapel environment variables are:

- `CHPL_RT_*`, e.g. ` CHPL_RT_OVERSUBSCRIBED`
  - Boolean values can be set as `0`/`1`, `true`/`false`, `t`/`f`. 
- `CHPL_LIB_PIC`
  - Boolean values can be set as `pic`/`none`

Should all of these env vars use the same values to represent on/off state?

**Comment by lydia-duncan** [comment]
When we renamed CHPL_LIBMODE to CHPL_LIB_PIC, we did consider `on` and `off` as its states but ultimately settled on `pic` and `none`. (source: [this comment thread](https://github.com/chapel-lang/chapel/pull/11327#discussion_r223881990)).  However, it looks like the reason behind rejecting `on` and `off` has been lost to the sands of time (I can see chats around that time, but I think the final discussion was verbal).

For further context, CHPL_LIBMODE was originally intended to have more states to it, but ultimately ended up being boiled down to the use or absence of `-fPIC`

**Comment by ben-albrecht** [comment]
@lydia-duncan - Thanks for the additional background info.

> For further context, CHPL_LIBMODE was originally intended to have more states to it, but ultimately ended up being boiled down to the use or absence of -fPIC

Do you think this is still true, or have we settled on an on/off state for this setting?

**Comment by lydia-duncan** [comment]
I'm comfortable with on/off state for this setting.  I think renaming the variable to this more specific name reflects that it has a more limited set of states and can't really think of what other ones it could have (though I am curious if @ronawho can remember any)

**Comment by ronawho** [comment]
> (though I am curious if @ronawho can remember any)

Nope, but this isn't something I'm really familiar with off the top of my head. Anytime I have to think about PIC/non-PIC, PIE,/non-PIE, and shared/static/whatever and other library modes I always end up having to look things up.
